ext/storage_sources/azure_store/azure_log_system.cpp

```diff
@@ -91,3 +92,12 @@ azure_log_system::log_debug_message(const std::string &message) const
 {
     log_verbose_message(WT_VERBOSE_DEBUG_1, message);
 }
+
+void azure_log_system::set_listener() const {
+    Azure::Core::Diagnostics::Logger::SetListener([this](auto lvl, auto msg) {
+        if (azure_to_wt_verbosity_level(lvl) <= _wt_verbosity_level)
+            _wt_api->err_printf(_wt_api, NULL, "%s", msg.c_str());
+        else
+            _wt_api->msg_printf(_wt_api, NULL, "%s", msg.c_str());
+    });
+}
```

Comment:
Missing newline at the end of the file here

Reply:
thanks for the pick up on my screen it shows an empty line but i've added another one for reassurance